projects/admin/src/app/service/record-routing.service.ts

```diff
@@ -202,8 +203,9 @@ export class RecordRoutingService {
           {
             key: 'patron_types',
             label: 'Patron Types',
+            canDelete: (record: any) => this.canDelete(record),
             component: PatronTypesBriefViewComponent,
-            preprocessRecordEditor: (data) => this.addLibrarianOrganisation(data)
```

Comment:
are you sure to remove this? I think patron_types need an organisation.

Reply:
No, it's a mistake.

Reply:
done